docusaurus/sidebars.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    docs: {
+        Welcome: ['motivation'],
+        'Getting Started': [
+            'getting-started/instructions',
+            {
+                type: 'category',
+                label: 'What\'s included',
+                items: [
+                    'getting-started/what-s-included/file-loaders',
```

Comment:
Change the name to `what-is-included` please 😅

Reply:
done.